[2023-12-10 19:40:06,585] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-28T00:00:00+00:00 [queued]>
[2023-12-10 19:40:06,987] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-28T00:00:00+00:00 [queued]>
[2023-12-10 19:40:07,040] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:40:07,075] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:40:07,088] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:40:07,735] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-03-28 00:00:00+00:00
[2023-12-10 19:40:07,761] {standard_task_runner.py:52} INFO - Started process 2633 to run task
[2023-12-10 19:40:07,928] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-03-28T00:00:00+00:00', '--job-id', '725', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpde4fz4ri', '--error-file', '/tmp/tmpyvtc5c3v']
[2023-12-10 19:40:07,960] {standard_task_runner.py:80} INFO - Job 725: Subtask segunda_tarea
[2023-12-10 19:40:09,561] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-28T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:40:12,738] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-03-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-28T00:00:00+00:00
[2023-12-10 19:40:12,744] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:40:12,774] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:40:13,066] {subprocess.py:85} INFO - Output:
[2023-12-10 19:40:13,168] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:40:13,183] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:40:13,798] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230328T000000, start_date=20231210T194006, end_date=20231210T194013
[2023-12-10 19:40:14,057] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:40:14,657] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
